otsdaq-mu2e/FEInterfaces/CFOFrontEndInterface_interface.cc

```diff
@@ -84,105 +87,155 @@ void CFOFrontEndInterface::registerFEMacros(void)
 					std::vector<std::string>{"Response"},  // namesOfOutput
-					1); 
+					1,
+					"*",
+					"This FE Macro measures the delay of a marker from ROCs. "
+					"Optionally, the delay can be measured over mutliple iterations with the <b>Loopback Test</B> Macro."
+	); 
 
 	registerFEMacroFunction(
 		"CFO Read",
 			static_cast<FEVInterface::frontEndMacroFunction_t>(
 					&CFOFrontEndInterface::ReadCFO),                  // feMacroFunction
 					std::vector<std::string>{"address"},  // namesOfInputArgs
 					std::vector<std::string>{"readData"},
-					1);  // requiredUserPermissions
+					1,  // requiredUserPermissions
+					"*", 
+					"Read from the CFO Memory Map.\n\n"
+					"Parameters:\n"
+					"\taddress (uint16_t): Address in Memory Map.\n"
+	); 
 
 	registerFEMacroFunction(
 		"Select Jitter Attenuator Source",
 			static_cast<FEVInterface::frontEndMacroFunction_t>(
 					&CFOFrontEndInterface::SelectJitterAttenuatorSource),
 				        std::vector<std::string>{"Source (0 is Local oscillator, 1 is RTF Copper Clock)","DoNotSet"},
 						std::vector<std::string>{"Register Write Results"},
-					1);  // requiredUserPermissions
+					1, // requiredUserPermissions
+					"*",
+					"The Jitter Attenuator is used to remove jitter, or variation in the timing of signals. "
+					"Select the source to subject the jitter attenuator: a local oscilator on the CFO or RTF.\n"
+					"RTF (RJ45 Timing Fanout) is a separate board to alleviate jitter accumulation. <b>Not all DTCs are connected to RTF</b>."
+	); 
 
 	registerFEMacroFunction(
 		"Reset Runplan",
 			static_cast<FEVInterface::frontEndMacroFunction_t>(
 					&CFOFrontEndInterface::ResetRunplan),                  // feMacroFunction
 					std::vector<std::string>{},  // namesOfInputArgs
 					std::vector<std::string>{},
-					1);  // requiredUserPermissions					
+					1,   // requiredUserPermissions					
+					"*",
+					"Resets the Event Building run plan by setting the reset bit (27) to true on the <b>CFO Control Register</b>."
+	);
 
 	registerFEMacroFunction(
 		"Compile Runplan",
 			static_cast<FEVInterface::frontEndMacroFunction_t>(
 					&CFOFrontEndInterface::CompileRunplan),                  // feMacroFunction
-					std::vector<std::string>{},//"Input Text Run Plan", "Output Binary Run File"},  // namesOfInputArgs
+					std::vector<std::string>{"Input Text File", "Output Binary File"},//"Input Text Run Plan", "Output Binary Run File"},  // namesOfInputArgs
 					std::vector<std::string>{"Result"},
-					1);  // requiredUserPermissions	
+					1,    // requiredUserPermissions	
+					"*" /* allowedCallingFEs */,
+					"This FE Macro compiles the CFO run plan to a binary file. You must compile before running <b>Set Runplan</b> "
+					"which downloads the binary run plan to the CFO.\n\nDefault text run plan: /mu2e_pcie_utils/cfoInterfaceLib/Command.txt\nDefault binary run plan: /mu2e_pcie_utils/cfoInterfaceLib/Command.bin" /* feMacroTooltip */
```

Comment:
Proposed change: add "srcs" to the start of the default path ==> e.g. "srcs/mu2e_pcie_utils/cfoInterfaceLib/Command.txt"